extensions/amp-timeago/0.1/amp-timeago.js

```diff
@@ -67,6 +67,15 @@ export class AmpTimeAgo extends AMP.BaseElement {
     }
   }
 
+  /** @override */
+  mutatedAttributesCallback(mutations) {
+    const datetime = mutations['datetime'];
+    if (datetime !== undefined) {
+      this.datetime_ = this.element.getAttribute('datetime');
+      this.setFuzzyTimestampValue_();
```

Comment:
should the part that sets `this.element.title` move to `setFuzzyTimestampValue_` as well to keep the title up to date?

Reply:
As discussed in #18441, the `title` attribute value is a bit difficult to resolve. I don't think there's anything we can do in `setFuzzyTimestampValue_()`. The only solution I can think of is to also allow the `title` attribute to be mutated with AMP bind.